crates/solidity/outputs/npm/package/package.json

```diff
@@ -55,7 +55,8 @@
         "x86_64-pc-windows-msvc",
         "x86_64-unknown-linux-gnu",
         "x86_64-unknown-linux-musl"
-      ]
+      ],
```

Comment:
If this is going to be a required property, should we add it to `testlang` as well?

Reply:
This is a required property when publishing, which we will never do for `testlang`. I think it's cleaner not to specify it here, what do you think?

Reply:
Actually, reading the code, `compiler.rs` only does that logic for release builds. Won't we need it for local development as well?
For example, if someone is building Slang on a linux machine, to debug the VS Code extension.

Reply:
The problem we're trying to solve here is to target correct GLIBC for distribution/publishing - a lot of different systems will attempt to load this library, so we need to make sure we're as compatible with different systems as possible.

For the development, the default behaviour on GNU in general when host-compiling is to target the (host) system GLIBC, as it's explained in OP and in the doc comment for the main routine. This is done to be compatible with the building host by definition, so the library will be compatible in the local development scenario. In fact, I'm daily-driving and using a Linux x64 GNU box to develop Slang + extension as well and it works as expected.

I don't think we need to apply the different GLIBC targeting logic to the debug pipeline. It only makes sense when we're distributing the binary outside, to a different system. For example, a debug build of the library is about ~47MB but the packaged one is ~4.8MB, so it seems unfeasible to officially support a "debug" channel; this seems like an orthogonal discussion whether we want to have something like canary/nightly/development release channel.

Reply:
Sounds good. Thanks!